test/unit/mdc-select/mdc-select.test.js

```diff
@@ -115,77 +82,153 @@ test('#get/setDisabled', () => {
+});
+
+test('#set value sets the value on the <select>', () => {
+  const {component, nativeControl} = setupTest();
+  component.value = 'orange';
+  assert.equal(nativeControl.value, 'orange');
+});
+
+test('#set value sets non value to empty string on <select>', () => {
+  const {component, nativeControl} = setupTest();
+  component.value = 'grape';
+  assert.equal(nativeControl.value, '');
+});
+
+test('#initialSyncWithDOM sets the selected index if an option has the selected attr', () => {
+  const fixture = bel`
+    <div class="mdc-select">
+      <div class="mdc-select__label">Pick a Food Group</div>
+      <div class="mdc-select__bottom-line"></div>
+      <select class="mdc-select__native-control">
+        <option value="orange">
+          Orange
+        </option>
+        <option value="apple" selected>
+          Apple
+        </option>
+      </select>
+    </div>
+  `;
+  const component = new MDCSelect(fixture, /* foundation */ undefined);
   assert.equal(component.selectedIndex, 1);
 });
 
-test('#initialSyncWithDOM disables the menu if aria-disabled="true" is found on the element', () => {
-  const fixture = getFixture();
-  fixture.setAttribute('aria-disabled', 'true');
+test('#initialSyncWithDOM disables the select if the disabled attr is found on the <select>', () => {
+  const fixture = bel`
+    <div class="mdc-select">
+      <div class="mdc-select__bottom-line"></div>
+      <select class="mdc-select__native-control" disabled>
+        <option value="orange">
+          Orange
+        </option>
+        <option value="apple" selected>
+          Apple
+        </option>
+      </select>
+    </div>
+  `;
   const component = new MDCSelect(fixture);
-  assert.isOk(component.disabled);
+  assert.isTrue(component.disabled);
 });
 
 test('adapter#addClass adds a class to the root element', () => {
   const {component, fixture} = setupTest();
   component.getDefaultFoundation().adapter_.addClass('foo');
-  assert.isOk(fixture.classList.contains('foo'));
+  assert.isTrue(fixture.classList.contains('foo'));
 });
 
 test('adapter#removeClass removes a class from the root element', () => {
   const {component, fixture} = setupTest();
   fixture.classList.add('foo');
   component.getDefaultFoundation().adapter_.removeClass('foo');
-  assert.isNotOk(fixture.classList.contains('foo'));
+  assert.isFalse(fixture.classList.contains('foo'));
+});
+
+test('does not create label', () => {
+  const fixture = bel`
+    <div class="mdc-select">
+      <div class="mdc-select__bottom-line"></div>
+      <select class="mdc-select__native-control">
+        <option value="orange">
+          Orange
+        </option>
+        <option value="apple" selected>
+          Apple
+        </option>
+      </select>
+    </div>
+  `;
+  const component = new MDCSelect(fixture);
+  component.getDefaultFoundation().adapter_.floatLabel('foo');
+  assert.equal(component.label_, undefined);
+});
+
+test('does not create bottomLine element', () => {
```

Comment:
See previous comment

Reply:
I'm going to wrap the adapter methods like so:
```
activateBottomLine: () => {
        if (this.bottomLine_) {
          this.bottomLine_.activate();
        }
      },
      deactivateBottomLine: () => {
        if (this.bottomLine_) {
          this.bottomLine_.deactivate();
        }
      },```